src/protocol/attribution/accumulate_credit.rs

```diff
@@ -124,7 +128,7 @@ pub async fn accumulate_credit<F: Field>(
             .into_iter()
             .enumerate()
             .for_each(|(i, (credit, stop_bit))| {
-                credits[i] = &credits[i] + &credit;
+                credits[i] = credits[i].clone() + &credit;
```

Comment:
I don't understand why this changed.

Reply:
Compiler cries because moving out of the vector issue.

Reply:
But why did it work before?